scripts/apps/authoring-react/toolbar/highlights-management.tsx

```diff
@@ -0,0 +1,100 @@
+import {sdApi} from 'api';
+import {dispatchInternalEvent} from 'core/internal-events';
+import {Card} from 'core/ui/components/Card';
+import {Spacer} from 'core/ui/components/Spacer';
+import {gettext} from 'core/utils';
+import React from 'react';
+import {IArticle, IHighlight} from 'superdesk-api';
+import {Button, Heading, IconButton, Text} from 'superdesk-ui-framework/react';
+
+interface IProps {
+    article: IArticle;
+    closePopup(): void;
```

Comment:
Does this component need to know whether it's being rendered inside a popup or not? It doesn't look like it to me. Maybe we could call it simply "close"